packages/inter-protocol/src/vaultFactory/vault.js

```diff
@@ -772,6 +775,25 @@ export const prepareVault = (baggage, marshaller, zcf) => {
           helper.updateUiState();
         },
 
+        /**
+         * Called by manager when collateral wasn't sold at auction. A fee was
+         * charged against collateral, but the debt is restored and the vault is
+         * active again.
+         */
+        reactivate() {
+          const { state, facets } = this;
+
+          const { helper } = facets;
+          // helper.updateDebtSnapshot(
+          //   // liquidated vaults retain no debt
+          //   AmountMath.makeEmpty(helper.debtBrand()),
+          // );
```

Comment:
no longer true, IIUC
```suggestion
          // helper.updateDebtSnapshot(
          //   // liquidated vaults retain no debt
          //   AmountMath.makeEmpty(helper.debtBrand()),
          // );
```

Reply:
gone